### PR TITLE
PCM-7691: Added one time bindings and pagination for case details performance

### PR DESCRIPTION
**Pull request by jeudy100**
@praniredhat @vrathee @anujsi please review

### CODE REVIEW

**Comment by vrathee** [comment]
Reviewed!

File: app/cases/views/discussionSection.jade [hunk 1]
```diff
@@ -26,139 +26,136 @@ ul#tab_list.nav.nav-tabs(role="tablist")
...
+        .spinner(ng-if="AttachmentsService.loading")
+        .row.row-very-short.case-discussion(ng-if="!AttachmentsService.loading")
             .col-xs-12
                 #top-discussion-section
                 .row
                     .col-sm-4
-                        rha-searchbox(placeholder='"Search Discussion"', class-name='"padding-searchbox"', ng-if="DiscussionService.allDiscussionElements().length > 0")
-                        span(ng-if="DiscussionService.allDiscussionElements().length > 0") Showing {{PaginationService.firstItemNumberShownOnThePage('discussionSection', DiscussionService.discussionElements.length)}}-{{PaginationService.lastItemNumberShownOnThePage('discussionSection', DiscussionService.discussionElements.length)}} of {{DiscussionService.discussionElements.length}} results
+                        rha-searchbox(placeholder='searchDiscussionString', class-name='"padding-searchbox"', ng-if="DiscussionService.allDiscussionElements().length > 0")
+                        span(ng-if="DiscussionService.allDiscussionElements().length > 0") {{showingResultsString()}}
```
Comment (praniredhat) [inline app/cases/views/discussionSection.jade]: Please revert placeholder value.

File: app/cases/views/discussionSection.jade [hunk 2]
```diff
@@ -26,139 +26,136 @@ ul#tab_list.nav.nav-tabs(role="tablist")
...
+                    article.comment.article-comment(ng-if="DiscussionService.isPublicComment(element)")
                         i.fa.fa-comments-o
                         h4.meta
-                            {{'Message'|translate}}&nbsp;({{element.created_by_type}})
-                            a.pull-right(ng-if="showJumpToCommentId == element.id" ng-click="jumpToComment(element.id)", ng-href='#/case/{{CaseService.kase.case_number}}?commentId={{element.id}}') Jump to comment
+                            {{::'Message'|translate}}&nbsp;({{::element.created_by_type}})
+                            a.pull-right(ng-if="ifJumpToCommentId == element.id" ng-click="jumpToComment(element.id)", ng-href='#/case/{{CaseService.kase.case_number}}?commentId={{::element.id}}') Jump to comment
                         .byline
```
Comment (praniredhat) [inline app/cases/views/discussionSection.jade]: ifJumpToCommentId => showJumpToCommentId

Comment (praniredhat) [reply app/cases/views/discussionSection.jade]: as well no change is required in `ng-href='#/case/{{CaseService.kase.case_number}}?commentId={{::element.id}}'`

File: app/cases/views/discussionSection.jade [hunk 3]
```diff
@@ -26,139 +26,136 @@ ul#tab_list.nav.nav-tabs(role="tablist")
...
     #pane2.tab-pane(ng-class='{"active": attachments}')
         #top-attachments-section
         .row
             .col-md-4
                 rha-searchbox(placeholder='"Search Attachments"', class-name='"padding-searchbox"', ng-if="AttachmentsService.originalAttachments.length > 0")
-                span(ng-if="AttachmentsService.originalAttachments.length > 0") Showing {{PaginationService.firstItemNumberShownOnThePage('attachmentsSection', DiscussionService.attachments.length)}}-{{PaginationService.lastItemNumberShownOnThePage('attachmentsSection', DiscussionService.attachments.length)}} of {{DiscussionService.attachments.length}} results
+                span(ng-if="AttachmentsService.originalAttachments.length > 0") ifing {{PaginationService.firstItemNumberifnOnThePage('attachmentsSection', DiscussionService.attachments.length)}}-{{PaginationService.lastItemNumberifnOnThePage('attachmentsSection', DiscussionService.attachments.length)}} of {{DiscussionService.attachments.length}} results
             .col-md-8
```
Comment (praniredhat) [inline app/cases/views/discussionSection.jade]: ifing => Showing

**Comment by praniredhat** [comment]
@jeudy100 Overall looks good, Please double check for if-show replacement. These changes look safe to me, you can merge to master if locally you have tested Comments section thoroughly.

**Pull request merged**